provider/pkg/provider/drift_schedules.go

```diff
@@ -32,7 +32,9 @@ func (i *PulumiServiceDriftScheduleInput) ToPropertyMap() resource.PropertyMap {
 	propertyMap := StackToPropertyMap(i.Stack)
 
 	propertyMap["scheduleCron"] = resource.NewPropertyValue(i.ScheduleCron)
-	propertyMap["autoRemediate"] = resource.NewPropertyValue(i.AutoRemediate)
+	if i.AutoRemediate != nil {
+		propertyMap["autoRemediate"] = resource.NewPropertyValue(*i.AutoRemediate)
+	}
```

Comment:
You should be able to avoid doing this by setting the default in the [schema](https://github.com/pulumi/pulumi-pulumiservice/blob/main/provider/cmd/pulumi-resource-pulumiservice/schema.json#L1110-L1113) like some [properties in deploymentSettings](https://github.com/pulumi/pulumi-pulumiservice/blob/main/provider/cmd/pulumi-resource-pulumiservice/schema.json#L303)

Reply:
Good point, didn't realize default values existed in schema. Updated the PR to use default values, but sadly then had to adjust Diff logic, as it turns out in Diff's GetNews() doesn't provide default values, so second and beyond pulumi up would always show diffs

Reply:
Huh, I find that surprising. We use a similar pattern in deploymentSettings and don't have to do this in the diff function. Looking at the generated SDKs it would be very surprising if this didn't work since they all provide the defaults if they're not specified. Which SDK were you testing with?

Reply:
Wondering if maybe the SDK wasn't regen'd before you tested.

Reply:
I used Dotnet SDK
I ran pulumi up on schedules without the booleans, and they were filled with false successfully (indicating that the schema change worked), but when I ran pulumi up again it showed the diff.
I will try to clean SDK and regenerate to make sure before I merge, just in case

Reply:
Confirmed by cleaning SDK, building this branch -> no change in diff, then removed explicit setting to false -> diff shows up. Sounds kind of like a bug